Test failure: JIT/opt/Multiply/MultiplyLongOps/MultiplyLongOps.cmd

arm64, jitstress_isas_nohwintrinsic

https://dev.azure.com/dnceng-public/public/_build/results?buildId=416992&view=ms.vss-test-web.build-test-results-tab&runId=9130856&paneView=debug

```
set DOTNET_TieredCompilation=0
set DOTNET_EnableHWIntrinsic=0
```

```
21:10:14.673 Running test: JIT/opt/Multiply/MultiplyLongOps/MultiplyLongOps.cmd

Assert failure(PID 848 [0x00000350], Thread: 7956 [0x1f14]): Assertion failed 'compiler->compIsaSupportedDebugOnly(HWIntrinsicInfo::lookupIsa(intrin.id))' in 'MultiplyLongOpsTest:smnegl_single_cast(int,int,long)' during 'Generate code' (IL size 15; hash 0xe1d89229; FullOpts)

    File: D:\a\_work\1\s\src\coreclr\jit\hwintrinsiccodegenarm64.cpp Line: 210
    Image: C:\h\w\B1000A13\p\corerun.exe

__tmp0_MultiplyLongOps.cs:19:11: error: ARM64: expected string not found in input
 //ARM64: {{^ *}} smaddl {{x[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}, {{x[0-9]+}}{{$}}
          ^
__jit_disasm.out:23:73: note: scanning from here
; BEGIN METHOD MultiplyLongOpsTest:smaddl_single_cast(int,int,long,long)
                                                                        ^
__jit_disasm.out:27:2: note: possible intended match here
 stp x19, x20, [sp, #0x10]
 ^

Input file: __jit_disasm.out
Check file: __tmp0_MultiplyLongOps.cs

-dump-input=help explains the following input dump.
...
... lots more ...
```

New test from https://github.com/dotnet/runtime/pull/91886

@c272 @kunalspathak

--- a/src/tests/JIT/opt/Multiply/MultiplyLongOps.csproj
+++ b/src/tests/JIT/opt/Multiply/MultiplyLongOps.csproj
@@ -14,5 +14,6 @@
 
     <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
     <CLRTestEnvironmentVariable Include="DOTNET_JITMinOpts" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_EnableHWIntrinsic" Value="1" />
   </ItemGroup>
 </Project>

